app/models/location.py

```diff
@@ -0,0 +1,26 @@
+from .. import db
+
+
+class ZIPCode(db.Model):
+    __tablename__ = 'ZIPCodes'
```

Comment:
I believe that tablenames are usually all-lowercase. If that's correct, this should probably be `zipcodes` or `zip_codes`? @aharelick?